### PR TITLE
Quick fixes for EDFM discretization

### PR DESCRIPTION
**Pull request by karimifard**
- Calculation of average distance to fracture.
- Correct contact area between fracture and matrix
- Consistency between DFM and EDFM

Rebaseline PR: https://github.com/GEOS-DEV/integratedTests/pull/4

### CODE REVIEW

File: src/coreComponents/finiteVolume/EmbeddedSurfaceToCellStencil.hpp [hunk 1]
```diff
@@ -205,8 +205,12 @@ EmbeddedSurfaceToCellStencilWrapper::
   localIndex const esr1 =  m_elementSubRegionIndices[iconn][1];
   localIndex const ei1  =  m_elementIndices[iconn][1];
 
+  // DFM does not use fracture permeability but here it is used
+  // we impose t1 = t0 to be consistent with DFM implementation for comparison
+  // TODO add the contribution of fracture for both DFM and EDFM
   real64 const t0 = m_weights[iconn][0] * LvArray::tensorOps::l2Norm< 3 >( coefficient[er0][esr0][ei0][0] );
-  real64 const t1 = m_weights[iconn][1] * LvArray::tensorOps::l2Norm< 3 >( coefficient[er1][esr1][ei1][0] );
+  // real64 const t1 = m_weights[iconn][1] * LvArray::tensorOps::l2Norm< 3 >( coefficient[er1][esr1][ei1][0] );
+  real64 const t1 = t0;
```
Comment (CusiniM) [inline src/coreComponents/finiteVolume/EmbeddedSurfaceToCellStencil.hpp:213]: I suspect that this will make the impermeable cases fail.

Comment (karimifard) [reply src/coreComponents/finiteVolume/EmbeddedSurfaceToCellStencil.hpp:213]: Do we have impermeable cases with EDFM?

Comment (CusiniM) [reply src/coreComponents/finiteVolume/EmbeddedSurfaceToCellStencil.hpp:213]: Not many, but we do have a few that test the pedfm work:
- https://github.com/GEOS-DEV/GEOS/blob/develop/inputFiles/singlePhaseFlowFractures/fractureMatrixFlow_pedfm_impermeableFracture_smoke.xml
- https://github.com/GEOS-DEV/GEOS/blob/develop/inputFiles/multiphaseFlowFractures/deadoil_3ph_corey_pedfm_impermeableFault_smoke.xml

**Comment by wrtobin** [comment]
The tags indicate this requires a rebaseline but there doesn't appear to be an open PR on the integratedTests repo linked in the PR.

If this won't require a rebaseline please remove the tag.

**Comment by TotoGaz** [comment]
> The tags indicate this requires a rebaseline but there doesn't appear to be an open PR on the integratedTests repo linked in the PR.

Hello, we plan to run the integrated tests when it's our turn to be in the merge queue.
That way we think we would reduce the risk of conflicts. I personally usually deal with integrated tests that way.
Do you think we should revise the pattern?

**Comment by wrtobin** [comment]
@TotoGaz 

Actually yeah that is a totally logical way to do things to minimize merge hassles on the integratedTests, though I do think there is a lack of consistency about that across the various PRs.

**Comment by wrtobin** [comment]
@TotoGaz @CusiniM @karimifard 

The PR ahead of this in line has just had its rebaselines merged, so this is up next.

New rebaselines and a PR on the integratedTests repo for this PR can now be generated, just make sure to update your integratedTests submodule to the current head of develop for the integratedTests repo.

**Comment by CusiniM** [comment]
> @TotoGaz @CusiniM @karimifard
> 
> The PR ahead of this in line has just had its rebaselines merged, so this is up next.
> 
> New rebaselines and a PR on the integratedTests repo for this PR can now be generated, just make sure to update your integratedTests submodule to the current head of develop for the integratedTests repo.

Thanks @wrtobin , I am on it. I

**Pull request merged**